korangar/src/world/map/mod.rs

```diff
@@ -341,6 +341,10 @@ impl Map {
         const OFFSET: f32 = 1.0;
         const TILE_SIZE: f32 = 5.0;
 
+        if !(0 <= position.x && position.x < self.width && 0 <= position.y && position.y < self.height) {
```

Comment:
This can just be re-written as
```rust
if position.x >= self.width || position.y >= self.height {
```
since `position.x` and `position.y` are `usize`.

Looking at the code I am unsure why this check is necessary, since we only ever call `render_walk_indicator` if the case that the mouse is hovering a tile. What was your reasoning here?

Reply:
There is a bug that the game crash in get_tile(), when you enter Izlude airport and try to enter the flying ship, with cursor above the teleporting portal.